core/src/main/java/com/oath/oak/common/integer/OakIntSerializer.java

```diff
@@ -0,0 +1,33 @@
+package com.oath.oak.common.integer;
+
+import com.oath.oak.OakSerializer;
+
+import java.nio.ByteBuffer;
+
+public class OakIntSerializer implements OakSerializer<Integer> {
```

Comment:
Are these 4 classes of  Int/IntBuffer comparator/serializer needed only for tests or also used in the Druid code?

Reply:
My two cents: Those classes are not used in Druid, but can help a new user to start working with Oak. For example, Ohad Serfati needed just <String,String> and erroneously made a buggy serializer/deserializer, until he realized we have those classes in our test package, so he copy-pasted. Ohad opened an issue asking to take such classes out of test package. If I understand it correctly this is done here (in addition to other changes).